app/components/e-checkbox.vue

```diff
@@ -0,0 +1,282 @@
+<template>
+  <div :class="b(modifiers)">
+
+    <input
+      :aria-checked="checked ? 'true' : 'false'"
```

Comment:
Thought the same. But since it returns the value and not true/false if we bind multiple checkboxes to the same v-model, we need to set it manually. Otherwise we would get something like this: https://cl.ly/rWQe